pkg/opcua/ca/pass

```diff
@@ -0,0 +1 @@
+testpass
```

Comment:
?

Reply:
It's the password file. The original design is that: define the username & password in the device instance yaml and pass it by the configmap. I think it's dangerous and change to use the password file which is in local directory.